src/assessment.js

```diff
@@ -27,7 +27,7 @@ var q1 = /* your answer here */;
 //
 //     `['alderaan', 'deathStar', 'fire', 'global']`
 //
-var q2 = [/* your answer here */];
+var q2 = [/* your answer here */]; 'deathStar', 'fire'
```

Comment:
i really don't know!

On Wed, Nov 18, 2015 at 9:45 PM, Matt Brendzel <notifications@github.com>
wrote:

> In src/assessment.js
> <https://github.com/ga-wdi-boston/js-scope-context-assessment/pull/36#discussion_r45293905>
> :
>
> > @@ -27,7 +27,7 @@ var q1 = /* your answer here */;
> >  //
> >  //     `['alderaan', 'deathStar', 'fire', 'global']`
> >  //
> > -var q2 = [/* your answer here */];
> > +var q2 = [/* your answer here */]; 'deathStar', 'fire'
>
> Correct... but why didn't you put your answer in the place that says /*
> your answer here */ ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ga-wdi-boston/js-scope-context-assessment/pull/36/files#r45293905>
> .
>



-- 
Laurisa Neuwirth
Cellular: 617-610-4838
Me: @LaurisaNeuwirth <https://twitter.com/LaurisaNeuwirth>
LinkedIn profile <http://www.linkedin.com/in/laurisaneuwirth/>